specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/servers.json

```diff
@@ -420,6 +420,18 @@
         "minimalTlsVersion": {
           "description": "Minimal TLS version. Allowed values: '1.0', '1.1', '1.2'",
           "type": "string"
+        },
+        "publicNetworkAccess": {
+          "description": "Whether or not public endpoint access is allowed for this server.  Value is optional but if passed in, must be 'Enabled' or 'Disabled'",
```

Comment:
> Value is optional but if passed in, must be 'Enabled' or 'Disabled'
I think that documentation explaining this will be automatically written (when generating SDKs) based on the x-ms-enum property. However, I think that what "public endpoint access" is might not be sufficiently obvious to somebody who doesn't have context.

Reply:
OK I can discuss with Rohit as to what is better documentation for this after this round is merged in first